app.js

```diff
@@ -1,19 +1,19 @@
 'use strict';
 var express = require('express'),
-	fs = require('fs'),
-	JSON5 = require('json5'),
+    bodyParser = require('body-parser'),
 	app = express(),
-	jsonData;
+    messages = [];
 
-var questions = [];
+app.use(express.static('public'));
+app.use(bodyParser.urlencoded({ extended: false }));
 
-fs.readFile('./public/freebase_countries.json', 'utf8', function(err, data){
-    jsonData = JSON5.parse(data);
-    getDataFromJson(jsonData, questions);
-    console.log(questions);
+app.post('/messages', function(req, res) {
+	  messages.push(req.body.message);
```

Comment:
ideally yes. E.x. request might fail for some reason. Than you need to handle error on frontend.

Search for `error` on this page http://api.jquery.com/jquery.post/ and read about it. See if it makes sense. If you don't understand that now - it's also fine.